sway/desktop/render.c

```diff
@@ -1946,6 +1828,9 @@ void output_render(struct sway_output *output, struct timespec *when,
 		goto renderer_end;
 	}
 
+	pixman_region32_t extended_damage;
+	pixman_region32_init(&extended_damage);
+
 	if (output_has_opaque_overlay_layer_surface(output)) {
 		goto render_overlay;
```

Comment:
Should we copy the damage region into extended_damage here?

Reply:
Oh, GitHub doesn't show the line that I selected... I meant copying the damage here:
```c
if (output_has_opaque_overlay_layer_surface(output)) {
	goto render_overlay;
}
```